samples/bluetooth/peripheral_power_profiling/boards/nrf54l15dk_nrf54l15_cpuapp_auto_conn_advert.overlay

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+/ {
+	aliases {
+		/delete-property/ sw2;
+		/delete-property/ sw3;
+	};
+	/*
+	 * Redefine button0 to use RXD1 - P0.01
+	 * Thus, when sending character from host, there will be gpio interrupt,
+	 * the same as originally triggered by sw0 button.
+	 */
+	buttons {
+		compatible = "gpio-keys";
+		button0: button_0 {
+			gpios = <&gpio0 1 (GPIO_PULL_UP | GPIO_ACTIVE_LOW)>;
+			label = "Input 0";
+			zephyr,code = <INPUT_KEY_0>;
+		};
+	};
+};
+
+&pinctrl {
+	/*
+	 * Redefine: RXD -> P0.13 (button0)
+	 */
+	/omit-if-no-ref/ uart30_default: uart30_default {
+		group1 {
+			psels = <NRF_PSEL(UART_TX, 0, 0)>,
+				<NRF_PSEL(UART_RTS, 0, 2)>;
+		};
+		group2 {
+			psels = <NRF_PSEL(UART_RX, 0, 13)>,
+				<NRF_PSEL(UART_CTS, 0, 3)>;
+			bias-pull-up;
+		};
+	};
+
+	/omit-if-no-ref/ uart30_sleep: uart30_sleep {
+		group1 {
+			psels = <NRF_PSEL(UART_TX, 0, 0)>,
+				<NRF_PSEL(UART_RX, 0, 13)>,
+				<NRF_PSEL(UART_RTS, 0, 2)>,
+				<NRF_PSEL(UART_CTS, 0, 3)>;
+			low-power-enable;
+		};
+	};
+};
+
+/delete-node/ &button2;
+/delete-node/ &button3;
+
+&uart20 {
+	zephyr,pm-device-runtime-auto;
+};
+
+&uart30 {
```

Comment:
Is it necessary to enable uart30, is it not enough to simply redefine button 0?

Reply:
Not needed, to be removed